src/vstart.sh

```diff
@@ -877,10 +883,23 @@ EOF
             # designate a single CPU node $osd for osd.$osd
             extra_seastar_args="--smp 1 --cpuset $osd"
         fi
+        local osd_pid
         run 'osd' $osd $SUDO $CEPH_BIN/$ceph_osd \
             $extra_seastar_args $extra_osd_args \
-            -i $osd $ARGS $COSD_ARGS
+            -i $osd $ARGS $COSD_ARGS &
+        osd_pid=$!
+        if $parallel
+        then
+            osds_wait=$osd_pid
+        else
+            wait $osd_pid
+        fi
     done
+    if $parallel
+    then
```

Comment:
ditto.

Reply:
@tchaikov DNM because I have to fix exiting when some error occurs.